test/wallet-test.js

```diff
@@ -1911,14 +1913,25 @@ describe('Wallet', function() {
     const workers = new WorkerPool({ enabled });
     const wdb = new WalletDB({ network, workers });
     const name = 'satoshi';
+    const nameHash = rules.hashName(name);
     const value = 1e6;
+    const secondHighest = value - 1;
     const lockup = 2e6;
     const fee = 10000;
-    let wallet;
+    let finalizeBlock;
```

Comment:
Now that there are a lot of these `describe` block global variables, adding comments here could be helpful